app/views/internal/broadcasts/_form.html.erb

```diff
@@ -9,6 +9,10 @@
   <%= label_tag :type, "Type:" %>
   <%= select_tag "type_of", options_for_select(%w[Welcome Announcement], selected: @broadcast.type_of) %>
 </div>
+<div class="form-group">
+  <%= label_tag :banner_style, "Banner Style:" %>
+  <%= select_tag "banner_style", options_for_select(%w[default brand success warning error], selected: @broadcast.banner_style), include_blank: true %>
```

Comment:
the repetition of the styles reminds me we should put it in a contant in the model

Reply:
Ah, good call 👍

Reply:
Addressed in last commit!